stam.py

```diff
@@ -0,0 +1,49 @@
+import sys
+import os
+
+"""
+:return 
+True if the path is a file,
+otherwise return false
+arg:
+path- the path received as parameter to the script
+"""
+
+
+def correct_parameter(path):
+    if not os.path.isfile(path):
+        print("your file not foud")
+        return False
+
+    return True
+
+
+"""
+the function adds a first name at the beginning of each line
+and a last name to the end of each line.
+args:
+filename- the path to the original file
+newfilename- the path to the new file
+"""
+
+
+def add_name(filename, newfilename):
+    with open(filename, 'r') as file:
+        for line in file:
+            with open(newfilename, 'a') as newfile:
+                newfile.write(f"keren {line.strip()} kayrich\n")
+
+
+def main():
```

Comment:
add documentation to main, and describe there how you expect the arguments to look like. with an example.